scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/simulation/SimulationFacade.java

```diff
@@ -193,9 +193,13 @@ private gov.nasa.jpl.aerie.constraints.model.ActivityInstance convertToConstrain
   {
     final var startT = Duration.of(startTime.until(driverActivity.start(), ChronoUnit.MICROS), MICROSECONDS);
     final var endT = startT.plus(driverActivity.duration());
+    var activityWindow = Window.betweenClosedOpen(startT, endT);
+    if(startT.isEqualTo(endT)){
+      activityWindow = Window.between(startT, endT);
+    }
```

Comment:
If we need to choose one between `final` and `var`, we should choose `final` 😝 

```suggestion
    final Window activityWindow;
    if (startT.isEqualTo(endT)) {
      activityWindow = Window.between(startT, endT);
    } else {
      activityWindow = Window.betweenClosedOpen(startT, endT);
    }
```

EDIT: Of course we could have both `final` and `var` if we make it a one-liner:

```suggestion
    final var activityWindow = startT.isEqualTo(endT)
      ? Window.between(startT, endT)
      : Window.betweenClosedOpen(startT, endT);
```